com.github.phase1geo.minder.yaml

```diff
@@ -0,0 +1,84 @@
+id: com.github.phase1geo.minder
+sdk: org.gnome.Sdk
+runtime: org.gnome.Platform
+runtime-version: '3.36'
+base: io.elementary.BaseApp
+base-version: juno-19.08
+command: com.github.phase1geo.minder
+finish-args:
+  - --env=DCONF_USER_CONFIG_DIR=.config/dconf
+  - --filesystem=~/.config/dconf:ro
```

Comment:
I don't think it needs dconf access?

Reply:
Sure, @bilelmoussaoui this is leftovers from you previews old one work. I'll fix this :soon: